app/views/nodes/show.html.erb

```diff
@@ -50,6 +50,7 @@
       <% if @sorted_evidence.any? %>
         <div data-behavior="datatable-paths"
             data-table-destroy-url="<%= multiple_destroy_project_node_evidence_index_path(@node.project, @node) %>"
+            data-table-destroy-confirmation="Are you sure?&#10;&#10;Proceeding will delete the selected item(s) from the associated node."
```

Comment:
If this is the default message, why not move it to the JS (instead of Are you sure?), so we can clean up the DOM and only provide the `data-` when a custom message is required?

You can perhaps cut the "from the associated node." part to make it even more generic.

Reply:
Done!